frontend/js/require.config.js

```diff
@@ -27,6 +27,7 @@ require.config({
         "i18next-browser-language-detector": "libs/i18nextBrowserLanguageDetector.min",
         "jquery-i18next": "libs/jquery-i18next.min",
         "moment": "libs/moment-with-locales",
+        "Hls": "libs/mediaelement/renderers/hls",
```

Comment:
I notice that also, I left with the capital "H" because that is the convention usage in the ME player. I don't have a strong opinion on that, I will change it :)